backend/convert_to_protobuf.go

```diff
@@ -193,6 +193,10 @@ func (t ConvertToProtobuf) QueryDataRequest(req *QueryDataRequest) *pluginv2.Que
 // It will set the RefID on the frames to the RefID key in Responses if a Frame's
 // RefId property is an empty string.
 func (t ConvertToProtobuf) QueryDataResponse(res *QueryDataResponse) (*pluginv2.QueryDataResponse, error) {
+	if res == nil {
+		return nil, nil
+	}
```

Comment:
is this a valid situation? (no res, no err?)

Reply:
Probably not. I'll update